index.njk

```diff
@@ -2,19 +2,54 @@
 layout: 'layouts/base.njk'
 ---
 
-<h1>
-  <span class="eyebrow">Design Sprint Guide</span>
-  Design, prototype, and test ideas quickly.
-</h1>
+<main class="tbds-app-frame" "tbds-block-stack" "tbds-block-stack--bordered">
+  
+  <header class="tbds-app-frame__header  tbds-margin-block-6">
+    <section class="tbds-inline-stack tbds-inline-stack--gap-6 tbds-inline-size-80%">
+      <div class="tbds-inline-stack__item">
+        <h5 class="tbds-brand-red">Design Sprint Guide <br/>
```

Comment:
This `<h5>` appears a level above a `<h1>` which isn't valid HTML. Perhaps this does need to be a heading?